pkg/apis/vsphereproviderconfig/v1alpha1/vsphereclusterproviderconfig_types.go

```diff
@@ -51,6 +52,39 @@ type VsphereClusterProviderConfig struct {
 	VspherePassword         string `json:"vspherePassword,omitempty"`
 	VsphereServer           string `json:"vsphereServer"`
 	VsphereCredentialSecret string `json:"vsphereCredentialSecret,omitempty"`
+
+	// SSHAuthorizedKeys is a list of SSH public keys authorized to access
+	// deployed machines.
+	//
+	// These keys are added to the default user as determined by cloud-init
+	// in the images from which the machines are deployed.
+	//
+	// The default user for CentOS is "centos".
+	// The default user for Ubuntu is "ubuntu".
+	SSHAuthorizedKeys []string `json:"sshAuthorizedKeys,omitempty"`
+
+	// CAKeyPair is the key pair for ca certs.
+	CAKeyPair KeyPair `json:"caKeyPair,omitempty"`
```

Comment:
Doesn't love mixing up `ca` and `CA`. Can we name this `clusterCAKeyPair` or does that break consistency with CAPA?

Reply:
Hi @andrewsykim,

So I literally copied all but `SSHAuthorizedKeys` verbatim.

Reply:
Hi @andrewsykim,

The [CAPA bits](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/6b455f697449d6e00eab9a02b57d3d80c8284182/pkg/apis/awsprovider/v1alpha1/awsclusterproviderconfig_types.go#L46-L60).